Client/Ecosia/Extensions/BrowserViewController+Ecosia.swift

```diff
@@ -14,7 +14,7 @@ extension BrowserViewController: HomepageViewControllerDelegate {
 
 extension BrowserViewController: APNConsentViewDelegate {
     func apnConsentViewDidShow(_ viewController: APNConsentViewController) {
-        User.shared.markAPNConsentScreenAsShown()
+
     }
 }
```

Comment:
Can't we remove all of this since it's empty? I guess we don't even need the delegate anymore then

Reply:
Absolutely, my bad! Did note that, but completely slipped my mind throughout the needed work. Will remove, thanks!